src/main/java/com/artipie/http/HealthSlice.java

```diff
@@ -93,7 +94,7 @@ public Response response(final String line,
                 new Key.From(".system", "test"),
                 new Content.From("OK".getBytes(StandardCharsets.US_ASCII))
             ).thenApply(none -> true).exceptionally(ignore -> false);
-        } catch (final IOException ignore) {
+        } catch (final IOException | UncheckedIOException ignore) {
```

Comment:
@genryxy handling `UncheckedIOException` here is wrong. It covers only one special case, the `FakeStorage` you have introduced in this PR. You should catch `Exception` here, handling just `IOException` here was wrong.